todo-app/src/todo/controller.ts

```diff
@@ -0,0 +1,54 @@
+import {
+    Body,
+    Controller,
+    Delete,
+    Get,
+    Param,
+    ParseIntPipe,
+    Post,
+    Put,
+  } from '@nestjs/common';
+  import { TodoService } from './service';
+  import {
+    CreateTodoRequestDto,
+    GetTodoDateRequestDto,
+  } from './dto/request.dtos';
+  import { TodoResponseDto } from './dto/response.dtos';
+  
+  @Controller('todo')
+  export class TodoController {
+    constructor(private todoService: TodoService) {}
+  
+    // create todo
+    @Post()
+    createTodo(
+      @Body() requestDto: CreateTodoRequestDto,
+    ): Promise<TodoResponseDto> {
+      return this.todoService.createTodo(requestDto);
+    }
+  
+    // get all todos for the username̦
+  
+    @Get('/:userName')
+    getTodoByUsername(
```

Comment:
- since you are getting a list, the method name should reflect that
- getTodosByUsername would be a better name